Xamarin.Forms.Core.UnitTests/ImageSourceTests.cs

```diff
@@ -74,10 +83,19 @@ public void TestImplicitStringUriConversion ()
 		}
 
 		[Test]
+		public void TestImplicitUriConversionWhenNull()
```

Comment:
This is the companion test which was previously failing, but now passes. Again, I don't agree with the behavior, but I made it match the existing behavior for `null` strings.